CommonTools/ParticleFlow/python/pfCandidatesBadHadRecalibrated_cfi.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
with the fillDescriptions defined, this file is superfluous and the system-made file should be used instead.